libs/agno/agno/api/schemas/evals.py

```diff
@@ -0,0 +1,24 @@
+from enum import Enum
+from typing import Any, Dict, Optional
+
+from pydantic import BaseModel
+
+
+class EvalType(str, Enum):
+    ACCURACY = "accuracy"
+    PERFORMANCE = "performance"
+    RELIABILITY = "reliability"
+
+
+class EvalRunCreate(BaseModel):
+    """Data sent to the API to create an evaluation run"""
+
+    agent_id: Optional[str] = None
+    user_id: Optional[str] = None
```

Comment:
Would you know the user_id? You can't know it at this point

Reply:
Misunderstood agent.user_id, I thought both the user and workspace ids were clear if having a contextual api key. Will refactor the data model and its consumers